.gitignore

```diff
@@ -3,3 +3,5 @@ coverage
 release-out
 .certs
 .tmp
+*.tmp
+.vscode
```

Comment:
We should probably not add (at least) the `.vscode` here; probably add a line similar to https://github.com/moby/moby/blob/1efda78f2bccb7b5654e6d02131ff2b1260b2663/.gitignore#L1-L3

We generally try to avoid adding editor/IDE-specific files, as there's _many_ possible options there, and those can be added to either a global `.gitignore`, or on a per-repository situation (using `.git/info/exclude`)

Reply:
fixed